src/test/kotlin/com/terraformation/backend/tracking/event/RateLimitedT0DataAssignedEventTest.kt

```diff
@@ -288,6 +288,61 @@ class RateLimitedT0DataAssignedEventTest {
     }
   }
 
+  @Nested
+  inner class CombineSiteTempSetting {
+    @Test
+    fun `combine keeps existing as previous value`() {
+      val existingEvent = event(previousSiteTempSetting = true, newSiteTempSetting = false)
+
+      val newEvent = event(previousSiteTempSetting = false, newSiteTempSetting = true)
+
+      assertEquals(
+          event(previousSiteTempSetting = true, newSiteTempSetting = true),
```

Comment:
In this case, would it be better to null out both values so it doesn't show up as a change at all?